### PR TITLE
add test for OCF different compression

### PR DESCRIPTION
**Pull request by Jeffxz**
Add test for https://github.com/w3c/epub-tests/issues/185

Used bzip2 for compression

```
zip -rX -Z bzip2 ../ocf-zip-comp.epub mimetype META-INF EPUB
  adding: mimetype (stored 0%)
  adding: META-INF/ (stored 0%)
  adding: META-INF/container.xml (bzipped 11%)
  adding: EPUB/ (stored 0%)
  adding: EPUB/content_001.xhtml (bzipped 19%)
  adding: EPUB/nav.xhtml (bzipped 22%)
  adding: EPUB/package.opf (bzipped 46%)
```

Extracted folder was tested by epubcheck. Zipped file raised compression error via epub check

### CODE REVIEW

**Comment by Jeffxz** [comment]
Will send PR to epub-specs after this one got merged

File: tests/ocf-zip-comp/EPUB/package.opf [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Open Container Format</dc:coverage>
+    <dc:creator>Zheng Xu</dc:creator>
+    <dc:date>2022-07-30</dc:date>
+        <dc:description>MUST treat any OCF ZIP container that uses compression techniques other than Deflate as in error.</dc:description>
+    <dc:identifier id="pub-id">ocf-zip-comp</dc:identifier>
```
Comment (iherman) [inline tests/ocf-zip-comp/EPUB/package.opf:8]: Minor knit: can you align the entry to the left with the rest of the items? Thx.

File: tests/ocf-zip-comp/EPUB/content_001.xhtml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>OCF ZIP container uses compression techniques other than Deflate</title>
+  </head>
+  <body>
+    <p>Test passes if reading system raises error when loading this epub file and this page of the book can not be accessed.</p>
```
Comment (iherman) [inline tests/ocf-zip-comp/EPUB/content_001.xhtml:6]: (I am being overly picky here...).

Very strictly speaking, we do not define what the RS' error behaviour is. It is not conceivable that a RS, whilst reporting an error, makes an attempt to display the content nevertheless. This should be acceptable (while making it clear that this _is_ an erroneous EPUB).

```suggestion
    <p>Test passes if reading system raises error when loading this epub file.</p>
```

Note that the same change should be made on the `ocf-zip-mult` test, too. My apologies not to have realized that before.

**Review by iherman — APPROVED** [review]

File: tests/ocf-zip-comp/EPUB/content_001.xhtml (hunk 2 above)
Comment (Jeffxz) [reply tests/ocf-zip-comp/EPUB/content_001.xhtml:6]: no problem. Thanks, I will make changes to both.

File: tests/ocf-zip-comp/EPUB/package.opf (hunk 1 above)
Comment (Jeffxz) [reply tests/ocf-zip-comp/EPUB/package.opf:8]: thnx, updated

**Comment by iherman** [comment]
@Jeffxz as far as I see, this is now done and can be merged, correct? If true, I will merge it and you should create a PR for the references in the spec.

Thanks!

**Comment by Jeffxz** [comment]
thanks @iherman. going to send PR to spec now.

**Pull request merged**